config/speechConfig.js.example

```diff
@@ -35,4 +35,8 @@ module.exports = {
     testChannel        : '@testpublishchannel', // a channel to make test publishes in
     testChannelPassword: 'password',  // password for the test channel
   },
+  api: {
+    apiUri: 'localhost',
```

Comment:
If I'm being pedantic, which I am wont to do, this is more like a host than a URI. All of the parts together comprise the URI.

Here's a reference (this terminology is fairly universal, not google specific): https://www.mattcutts.com/blog/seo-glossary-url-definitions/

Reply:
I'm wont to do it correctly ;)